README.rst

```diff
@@ -218,6 +218,10 @@ In order to run the unit tests and to get a coverage report, four additional Pyt
   pip3 install --user pytest
   pip3 install --user coverage
 
+Most of the unit tests can also be run in parallel. This can be done by installing one additional package::
+
+  pip3 install --user pytest-xdist
+
```

Comment:
Could you please fix line 213 of this file in this PR?

It says "four additional Python packages should be installed", but they are not four.

Reply:
Fixed in 7c944fb

Reply:
Thanks!